src/Compiler/Dynamic/Conversion.cs

```diff
@@ -568,25 +568,24 @@ public static bool TryGetArrayConversion(int fromRank, int toRank, Type toElemen
 
         private class Pair
```

Comment:
This file should not be changed at all.  "Pair" is simply an unfortunate name choice for this class.  A better name would be ConvertedDelegate.

Reply:
[PR 39](https://github.com/dotnet/infer/pull/39) fixes this.